packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -707,6 +707,44 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for SystemData
+    ///
+
+    // RPC Code: RPC-SystemData-V1-01 and RPC-SystemData-V1-02
+    // Note that this rule is normally already covered when the common types Resource is used.
+    // Here we are still checking for usage of common types system data for the scenarios where
+    // RP teams have already defined Resource and are unable to switch to the common types resource
```

Comment:
```suggestion
    // RP teams have already defined Resource and are unable to switch to the common types Resource
```

or `resource`, but then let's make the capitalization consistent